frontend/javascripts/oxalis/model/bucket_data_handling/wkstore_adapter.js

```diff
@@ -145,23 +146,40 @@ export async function requestFromStore(
     createRequestBucketInfo(zoomedAddress, resolutions, fourBit, applyAgglomerates, version),
   );
 
-  return doWithToken(async token => {
-    const { buffer: responseBuffer, headers } = await Request.sendJSONReceiveArraybufferWithHeaders(
-      `${dataUrl}/data?token=${token}`,
-      {
-        data: bucketInfo,
-        timeout: REQUEST_TIMEOUT,
-      },
-    );
-    const missingBuckets = parseAsMaybe(headers["missing-buckets"]).getOrElse([]);
-
-    let resultBuffer = responseBuffer;
-    if (fourBit) {
-      resultBuffer = await decodeFourBit(resultBuffer);
-    }
-
-    return sliceBufferIntoPieces(layerInfo, batch, missingBuckets, new Uint8Array(resultBuffer));
-  });
+  try {
+    return doWithToken(async token => {
+      const stuff = await Request.sendJSONReceiveArraybufferWithHeaders(
```

Comment:
Maybe `response` or `bufferWithHeaders` or something along these lines instead of `stuff`?

Reply:
Or simply undo this and stick to the destructuring? I guess, this was only for debugging purposes?

Reply:
Indeed :see_no_evil: